hail/python/test/hail/utils/test_google_fs_utils.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+
+import hail as hl
+from hail.utils import *
+from hail.utils.java import Env
+from hail.utils.linkedlist import LinkedList
+from hail.fs.hadoop_fs import HadoopFS
+from ..helpers import *
+import os
+
+setUpModule = startTestHailContext
+tearDownModule = stopTestHailContext
+
+
+class Tests(unittest.TestCase):
+    BUCKET = os.environ.get("TEST_BUCKET_NAME", None)
+
+    @classmethod
+    def setUpClass(cls):
+        if cls.BUCKET is None:
```

Comment:
I forget my old comment, but I think this is clearer as a global variable.  I think I was just referring to HAIL_TEST_SERVICE_BACKEND_URL which you defined globally but used in only one place.